pkg/kubernetes/styles/sidebar.less

```diff
@@ -4,7 +4,7 @@
 
 
 .nav-sidebar {
-    background: #393f44;
+    background: var(--color-alert-success-text);
```

Comment:
This probably isn't the right variable? We should look at it in context. (Then again, it seems like it's using green for some reason? I don't know. Weird.)